test/verifiedTests/analysis/testPrint/refData_surfNet.txt

```diff
@@ -3,402 +3,402 @@ Met #1  13dpg[c], 3-Phospho-D-glyceroyl-phosphate, C3H4O10P2
 
 Consuming reactions: none
 Producing reactions:
-  #49  <a href="matlab: surfNet([], 'GAPD', 0, 'none', 0, 1, [], 0);">GAPD</a>, Bd: -1000 / 1000, glyceraldehyde-3-phosphate dehydrogenase
-<a href="matlab: surfNet([], 'g3p[c]', 0, 'none', 0, 1, [], 0);">g3p[c]</a> + <a href="matlab: surfNet([], 'nad[c]', 0, 'none', 0, 1, [], 0);">nad[c]</a> + <a href="matlab: surfNet([], 'pi[c]', 0, 'none', 0, 1, [], 0);">pi[c]</a> <=> <a href="matlab: surfNet([], '13dpg[c]', 0, 'none', 0, 1, [], 0);">13dpg[c]</a> + <a href="matlab: surfNet([], 'h[c]', 0, 'none', 0, 1, [], 0);">h[c]</a> + <a href="matlab: surfNet([], 'nadh[c]', 0, 'none', 0, 1, [], 0);">nadh[c]</a>  
-  #75  <a href="matlab: surfNet([], 'PGK', 0, 'none', 0, 1, [], 0);">PGK</a>, Bd: -1000 / 1000, phosphoglycerate kinase
-<a href="matlab: surfNet([], '3pg[c]', 0, 'none', 0, 1, [], 0);">3pg[c]</a> + <a href="matlab: surfNet([], 'atp[c]', 0, 'none', 0, 1, [], 0);">atp[c]</a> <=> <a href="matlab: surfNet([], '13dpg[c]', 0, 'none', 0, 1, [], 0);">13dpg[c]</a> + <a href="matlab: surfNet([], 'adp[c]', 0, 'none', 0, 1, [], 0);">adp[c]</a>  
+  #49  <a href="matlab: surfNet([], 'GAPD', 0, NaN, 0, 1, [], 0);">GAPD</a>, Bd: -1000 / 1000, glyceraldehyde-3-phosphate dehydrogenase, grRules: b1779
```

Comment:
Is this change backward compatible?

Reply:
It should be if the function is called in the intended way. This NaN or 'none' are supposed to be used for iterative internal calling only (during mouse clicking through the command window). And changing from 'none' to NaN is to avoid potential confusion when recognizing the parameter/value input keywords.

Reply:
Sounds good.